src/openms/source/ANALYSIS/QUANTITATION/TMTSixteenPlexQuantitationMethod.cpp

```diff
@@ -0,0 +1,220 @@
+    defaults_.setValue("channel_129C_description", "", "Description for the content of the 129C channel.");
+    defaults_.setValue("channel_130N_description", "", "Description for the content of the 130N channel.");
+    defaults_.setValue("channel_130C_description", "", "Description for the content of the 130C channel.");
+    defaults_.setValue("channel_131N_description", "", "Description for the content of the 131N channel.");
+    defaults_.setValue("channel_131C_description", "", "Description for the content of the 131C channel.");
+    defaults_.setValue("channel_132N_description", "", "Description for the content of the 132N channel.");
+    defaults_.setValue("channel_132C_description", "", "Description for the content of the 132C channel.");
+    defaults_.setValue("channel_133N_description", "", "Description for the content of the 133N channel.");
+    defaults_.setValue("channel_133C_description", "", "Description for the content of the 133C channel.");
+    defaults_.setValue("channel_134N_description", "", "Description for the content of the 134N channel.");
+
+    defaults_.setValue("reference_channel", "126", "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131N, 131C).");
+    defaults_.setValidStrings("reference_channel", TMTSixteenPlexQuantitationMethod::channel_names_);
+
+    defaults_.setValue("correction_matrix", ListUtils::create<String>("0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0"),
+                       "Correction matrix for isotope distributions (see documentation); use the following format: <-2Da>/<-1Da>/<+1Da>/<+2Da>; e.g. '0/0.3/4/0', '0.1/0.3/3/0.2'");
```

Comment:
@poshul why is the correction matrix all zero? 

@poshul it say *see documentation* but where is this actually documented what this matrix will do?

Reply:
do we have access to a TMT16 purity certificate which could use as a starting point to fill in some sensible values here?
Not sure what the deal with TMT is, but with Itraq it used to be pretty stable, so the user has to adapt the matrix only partially. Right now, all the numbers have to be typed in manually... this is quite cumbersome...

Reply:
There are a number of different batches for the tmt16plex (as well as the other TMT procedures). I have no experience running TMT experiments, @cbielow can you take a look and let me know if the varience between batches is significant or not?

Reply:
I think the danger here is that the user will try and use the tool with the default settings and then get a wrong result because the matrix was all zero. Also as @cbielow mentioned, typing in all the numbers is quite cumbersome and not a great user experience, but I dont really know how to get around that problem -- unless there is a reasonable value for the matrix or make sure to check that the matrix is not zero and explain to the user how to type in those numbers. Maybe we can provide the numbers in a file that we read in instead of using a parameter that is hard to type?

Reply:
I've gone through the lot data sheets for the different TMT types. For 10plex and 16plex, there is very little variation between lots, so I feel comfortable putting in a default set of values made up of the means from the different lots. For the 11plex and the 6plex on the other hand, there appears to be substantial variation, such that I am hesitant to include a mean. Any suggestions?